actions/rm-runner-ip/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -eu
+
+# remove the runner IP from the list of authorized IPs
+CUR_IPS=$(gcloud container clusters describe "${K8_CLUSTER}" --format json | \
```

Comment:
You could also get the ips with `gcloud container clusters describe "${K8_CLUSTER}"
--format="get(masterAuthorizedNetworksConfig.cidrBlocks[].cidrBlock)"`. Although this is more useful when you otherwise wouldn't need `jq`.

Reply:
This is neat, but it outputs a semicolon-separated list of IPs, not a comma separated JSON array of IPs.